docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -348,6 +350,34 @@ GET /_search
 \... will sort the composite bucket in descending order when comparing values from the `date_histogram` source
 and in ascending order when comparing values from the `terms` source.
 
+====== Missing bucket
+
+By default documents without a value for a given source are ignored.
+It is possible to include them in the response by setting `ignore_missing` to
```

Comment:
The option is `missing_bucket` below, is this a typo?

Reply:
Yes this is a typo, thanks